Content.Server/Atmos/EntitySystems/AtmosphereSystem.cs

```diff
@@ -65,8 +71,10 @@ public override void Initialize()
         _airtightQuery = GetEntityQuery<AirtightComponent>();
         _firelockQuery = GetEntityQuery<FirelockComponent>();
 
-        SubscribeLocalEvent<TileChangedEvent>(OnTileChanged);
+        // Store burnt tiles decals for the future use
+        _burntDecals = _prototypeManager.EnumeratePrototypes<DecalPrototype>().Where(x => x.Tags.Contains("burnt")).Select(x => x.ID).ToList();
```

Comment:
Doesn't work with prototype reloads, just add a prototypemanager sub and refresh it there.